apps/estuary/test/unit/message_handler_test.exs

```diff
@@ -0,0 +1,84 @@
+defmodule Estuary.MessageHandlerTest do
+  use ExUnit.Case
+
+  import Mox
+  import Assertions
+
+  alias Estuary.MessageHandler
+  alias SmartCity.TestDataGenerator, as: TDG
+  alias DeadLetter.Carrier.Test, as: Carrier
+
+  setup :set_mox_global
+  setup :verify_on_exit!
```

Comment:
i think you can remove all these `Mox` things (including the import above) since you don't seem to be using them anywhere in this file.